### PR TITLE
Add instructions for getting and updating the configuration in prod

### PR DESCRIPTION
**Pull request by arkadyan**
~~I can't remember all the details for the final steps of updating. We'll need to
flesh those out the next time we actually make an update.~~

### CODE REVIEW

**Comment by arkadyan** [comment]
@paulswartz I started instructions for as far as I could remember. Feel free to give me any changes or flesh out the rest if you remember the details, or otherwise I can finish this the next time we have an actual update to make.

File: README.md [hunk 1]
```diff
@@ -36,9 +35,28 @@ ApiChecker checks are run via a well defined and strict json configuration.
 
 A perodic task is configured by placing a json `array` of valid periodic task JSON objects in one of three files. Each file is loaded upon startup in `dev`, `test`, and `prod` environments, respectively:
 
-+ `./priv/dev_checks_config.json`
+- `./priv/dev_checks_config.json`
+
```
Comment (paulswartz) [inline README.md]: extra space?

File: README.md [hunk 2]
```diff
@@ -36,9 +35,28 @@ ApiChecker checks are run via a well defined and strict json configuration.
...
+1. Under `Container Definitions` expand the `api-checker` row
+1. Under `Environment Variables` you'll find the definition for `API_CHECKER_CONFIGURATION`
+
+##### Updating the configuration
+
+1. Follow the steps for getting the current configuration which get you to the `Task Definition` page for the current version of the task
+1. Click `Create new revision`
+1. Under `Container Definitions` click the `api-checker` container name
+1. Under the `Environment` section, enter your updated value for the `API_CHECKER_CONFIGURATION` key
+1. Click `Update`
```
Comment (paulswartz) [inline README.md]: There's another button to click after updating the environment.

**Comment by arkadyan** [comment]
@paulswartz I finished describing the remaining steps in the process and removed the extra line you spotted. This should be ready to go at this point.

**Comment by paulswartz** [comment]
Force merging since there aren't any holidays in the current rating.

**Pull request merged**